exporters/trace/src/main/java/com/google/cloud/opentelemetry/trace/TraceExporter.java

```diff
@@ -15,134 +15,108 @@
  */
 package com.google.cloud.opentelemetry.trace;
 
-import static com.google.api.client.util.Preconditions.checkNotNull;
-
-import com.google.api.gax.core.FixedCredentialsProvider;
-import com.google.api.gax.core.NoCredentialsProvider;
-import com.google.api.gax.grpc.GrpcTransportChannel;
-import com.google.api.gax.rpc.FixedTransportChannelProvider;
-import com.google.api.gax.rpc.HeaderProvider;
-import com.google.auth.Credentials;
-import com.google.auth.oauth2.GoogleCredentials;
-import com.google.cloud.trace.v2.TraceServiceClient;
-import com.google.cloud.trace.v2.TraceServiceSettings;
-import com.google.cloud.trace.v2.stub.TraceServiceStub;
-import com.google.common.collect.ImmutableMap;
-import com.google.devtools.cloudtrace.v2.AttributeValue;
-import com.google.devtools.cloudtrace.v2.ProjectName;
-import com.google.devtools.cloudtrace.v2.Span;
-import io.grpc.ManagedChannelBuilder;
+import com.google.cloud.ServiceOptions;
 import io.opentelemetry.sdk.common.CompletableResultCode;
 import io.opentelemetry.sdk.trace.data.SpanData;
 import io.opentelemetry.sdk.trace.export.SpanExporter;
 import java.io.IOException;
-import java.util.ArrayList;
 import java.util.Collection;
-import java.util.List;
-import java.util.Map;
+import java.util.Objects;
+import java.util.concurrent.atomic.AtomicReference;
+import java.util.logging.Level;
```

Comment:
We should probably use SLF4J rather than JUL. Thoughts @jsuereth ?

Reply:
Would defer to whatever is decided here. But is there some benefit here for SLF4J ?

IMO though SLF4J is helpful when there are a lot of logs and they needed some sort of formatting or structure applied to them. This would also require additional config.

For simple logging Java logger would be better.

Reply:
SLF4J is a logging facade, not a specific logging library. It allows the end-user of the application to decide which library to use (like Log4J, Logback, etc).